src/libraries/System.Collections/tests/Generic/PriorityQueue/PriorityQueue.Tests.cs

```diff
@@ -207,6 +242,16 @@ public void PriorityQueue_EmptyCollection_Peek_ShouldReturnFalse()
             Assert.Throws<InvalidOperationException>(() => queue.Peek());
         }
 
+        [Fact]
+        public void PriorityQueue_EmptyCollection_Remove_ShouldReturnFalse()
+        {
+            var queue = new PriorityQueue<string, string>();
+
+            Assert.False(queue.Remove(element: "element", out string removedElement, out string removedPriority));
+            Assert.Null(removedElement);
+            Assert.Null(removedPriority);
+        }
+
```

Comment:
Since the primary reported purpose of adding this method is to enable non-production scenarios involving implementing Djikstra's algorithm, it'd be good to have a test that does so.

Reply:
How does this look? https://github.com/dotnet/runtime/pull/93994/commits/5d2f0bfe44a2db15643033643677d466c87cd3b0